webapi/admin.go

```diff
@@ -197,12 +198,24 @@ func ticketSearch(c *gin.Context) {
 	})
 }
 
+//checkPasswordHash compare hash value of given AdminPass with AdminPass hash.
+func CheckPasswordHash(hash []byte, password string) bool {
+    err := bcrypt.CompareHashAndPassword(hash, []byte(password))
+    return err == nil
+
+}
 // adminLogin is the handler for "POST /admin". If a valid password is provided,
 // the current session will be authenticated as an admin.
 func adminLogin(c *gin.Context) {
 	password := c.PostForm("password")
+	hashedPass, err := db.GetAdminHash()
+	if err != nil {
+		log.Warnf("Not able to get Admin Hash: %w", err)
+	}
```

Comment:
Should this method return when this error is encountered? How about saving the password hash gotten from db when the program is started in a variable to be compared later, to avoid potentially hitting a db error that prevents login from succeeding?